draft-ietf-quic-recovery.md

```diff
@@ -398,8 +419,44 @@ A packet sent on an RTO alarm MUST NOT be blocked by the sender's congestion
 controller. A sender MUST however count these bytes as additional bytes in
 flight, since this packet adds network load without establishing packet loss.
 
+## Multiple Packet Number Space Optimizations {#optimizations}
+
+There are cases where one may be able to gain recovery information from
+acknowledgements of packets in another packet number space, but they rely
+on complex assumptions about the peer’s processing and acknowledgement
+algorithms.  Even those are unable to quickly recover from cases such as
+losing the client's Initial, but receiving the 0-RTT packets.  Below is
+an optimization using coalesced packets and implicit acknowledgements.
+
+### Coalesced Packets
+
+Despite loss recovery being separate for different packet number spaces,
+the ability to send a coalesced packet enables faster recovery with small,
+and sometimes no overhead.  The acknowledgement of a coalesced packet
+allows QUIC recovery to use early retransmit to determine if any prior
+packets in that space were lost without waiting for timeouts.
+
+This optimization is particularly useful when:
+
+ * Sending the client’s Initial, which must be padded to a full
+   sized packet, so the datagram typically has extra space to retransmit
+   some outstanding 0-RTT data.
+ * The clients sends 1-RTT data soon after the final TLS flight
+   (containing the client Finished) and can proactively retransmit the
+   final client flight with one or more 1-RTT packets.
+
+### Implicit Acknowledgements of Initial
+
+Initial data may be cancelled when packets at the Handshake level
+are received and processed, because that indicates all packets at
+Initial encryption have been received and processed by the peer.
+
```

Comment:
@ianswett: I think we should move this sxn to another PR. Arguably, we should remove the entire super-section. If not, I will try to rewrite the Coalesced Packets section for clarity